app/k9mail/src/main/res/xml/message_list_widget_info.xml

```diff
@@ -9,5 +10,6 @@
     android:previewImage="@drawable/message_list_widget_preview"
     android:resizeMode="horizontal|vertical"
     android:updatePeriodMillis="86400000"
-    android:widgetCategory="home_screen|keyguard">
+    android:widgetCategory="home_screen|keyguard"
+    tools:targetApi="jelly_bean_mr1">
```

Comment:
The widget is also available on versions prior to `jelly_bean_mr1`, so `targetApi` feels wrong. Those Android versions just ignore the unsupported attributes/values.